src/main/java/by/exadel/internship/dto/formDTO/FormRegisterDTO.java

```diff
@@ -68,4 +68,6 @@
 
     private List<TimeForCallDTO> timeForCallList;
 
+    private boolean confirmEmail = false;
```

Comment:
JsonProperty.Access.READ_ONLY - for avoiding setting this flag from UI.

```suggestion
   @JsonProperty( access = JsonProperty.Access.READ_ONLY )
    private boolean sendEmail = false;
```

Reply:
done